SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -256,7 +278,10 @@ void ASpatialFunctionalTest::FinishTest(EFunctionalTestResult TestResult, const
 
 		CurrentStepIndex = SPATIAL_FUNCTIONAL_TEST_FINISHED;
 		OnReplicated_CurrentStepIndex(); // need to call it in Authority manually
-		MulticastAutoDestroyActors(AutoDestroyActors);
+
+		MulticastAutoDestroyActors();
```

Comment:
The MutlicastAutoDestroyActors is no longer needed. In OnReplicated_CurrentStepIndex, there's where you know that a Test has finished, if the CurrentStepIndex == SPATIAL_FUNCTIONAL_TEST_FINISHED. Since this is already replicated to all Workers, then you simply have to check if they are servers and if so call DeleteActorsRegisteredForAutoDestroy()

Reply:
Sure, done.